src/main/java/io/vertx/core/impl/VertxImpl.java

```diff
@@ -1082,6 +1087,13 @@ void release() {
     }
   }
 
+  public class DefaultExecutorServiceFactory implements ExecutorServiceFactory {
```

Comment:
can we instead use `FixedThreadPoolExecutorServiceFactory` that seems to be the same class ?

Reply:
my bad, FixedThreadPoolExecutorServiceFactory is in test package, forget it.